schemas/security/SystemLog.schema

```diff
@@ -1,7 +1,7 @@
-{
+Log({
   User: { category: 'SystemUser', required: true },
   Category: { domain: 'Id', required: true },
   Timestamp: { domain: 'Timestamp', required: true },
   Operation: { domain: 'UserAccess', required: true },
   Ids: { domain: 'List' }
```

Comment:
Not related to this PR, but maybe it's worth introducing `List-type` (List<Id>)? Because now this list can possibly contain anything based on this schema.

Reply:
Here I mean `List` is a collection of Ids: set, list or array. We have domain, described in [/schemas/domains.schema#L18](https://github.com/metarhia/metaschema/blob/b3d86629fd3baad15fe2a7eeedfa6d74dd59c417/schemas/domains.schema#L18) but have no code "knowing" that :smile: @lundibundi